File: basic/OOP_Variable/4.rb
class C
  # attribute = value
  '''
  # set 없이 변수에 접근
  attr_reader :value
  # get없이 변수에 접근
  attr_writer :value
  '''
  # read/write 허용
  attr_accessor :value
  def initialize(v)
    @value = v
  end
  def show()
    p @value
  end
end

c1 = C.new(10)
p c1.value
c1.value = 20
p c1.value

File: basic/OOP_Variable/4.py
class C(object) :
    def __init__(self, v) :
# 변수 이름 앞에 __ 붙이면
# 외부에서 접근 안된다.
        self.__value = v
    def show(self) :
        print(self.__value)

c1 = C(10)
# print(c1.__value)
c1.show()
